src/components/asset-panel/selector.jsx

```diff
@@ -55,7 +56,11 @@ const Selector = props => {
 
 Selector.propTypes = {
     buttons: PropTypes.arrayOf(PropTypes.shape({
-        text: PropTypes.string.isRequired,
+        message: PropTypes.shape({
+            id: PropTypes.string.isRequired,
+            defaultMessage: PropTypes.string,
+            description: PropTypes.string
+        }),
```

Comment:
After talking with @rschamp and @chrisgarrity, we should 
- [ ] change this prop to take a `FormattedMessage` (how to do proptypes for that? PropTypes.node? PropTypes.shape({type: FormattedMessage})?)
- [ ] change `defineMessage` in sound/costume tab to just be `FormattedMessage` directly